0miner

```diff
@@ -190,11 +190,11 @@ then
   pm_manifest=${mpath}/${!xversion}/nvoc-miner.json
   
   # parse device enable params
-  dev_enable_arg="$(jq -r .devlist_argument ${pm_manifest})"
+  dev_enable_arg=$(eval $(jq -r .devlist_argument ${pm_manifest}))
```

Comment:
this doesn't work, .devlist_argument does not contain an executable command, therefore you get somewhat equivalent to this `dev_enable_arg=$(eval --cuda_devices)` which does gives token error with eval -- as long as -- is not something evaluable.
As a side effect the if block does not get executed properly.

Reply:
I can give it a shot without this in place.  I'm not entirely sure what's going on here, but I know that when I ran my tests, this was changed and miners weren't starting.  This is basically me just putting it back to what it was and then things started running.